### DISCUSSION TITLE
Multiple sequencing runs of the same sample

### DISCUSSION THREAD

**Discussion by gandreeva18** [stuart-lab/signac#1496] [Q&A] [answered]
Good morning, 

I have several sequencing runs of the same sample, to increase sequencing depth. To analyze these samples, should I merge them and analyze as one sample, or should I integrate them as separate samples?

Thanks,
GS

**Comment by timoast** [comment, marked as answer]
Since these are additional reads for the same cells, you want to combine the reads at the mapping step. Merging or integration is for separate datasets (separate sets of cells).

See the cellranger documentation on how to handle multiple sequencing runs:
https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/using/count
https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/using/aggr
